### DISCUSSION TITLE
为何运行npm run build并未生成静态网站的根目录？

### DISCUSSION THREAD

**Discussion by briteming** [MaLuns/vitepress-theme-async#4] [General]
hi.

12799@DESKTOP-B6LK9IO MINGW64 ~/at-vp-site
$ npm run build

> vitepress-blog@1.0.0 build
> vitepress build ./

  vitepress v1.4.1

✓ building client + server bundles...
✓ rendering pages...

💖 Vitepress-Theme-Async. 0.0.20  Guide: https://vitepress-theme-async.imalun.com

build complete in 28.50s.

12799@DESKTOP-B6LK9IO MINGW64 ~/at-vp-site
$

12799@DESKTOP-B6LK9IO MINGW64 ~/at-vp-site
$ ls
about.md  node_modules/      package.json  public/
index.md  package-lock.json  posts/

12799@DESKTOP-B6LK9IO MINGW64 ~/at-vp-site
$ ls public
favicon.svg

在public/里面，没看到有index.html文件

**Comment by MaLuns** [comment]
又可以复现 demo 吗

**Comment by brightmann** [comment]
hi.

$ npm run dev
> vitepress-blog@1.0.0 dev
> vitepress dev ./

  vitepress v1.4.1

  ➜  Local:   http://localhost:5173/
  ➜  Network: use --host to expose
  ➜  press h to show help

我运行npm run dev，然后在浏览器里访问http://localhost:5173，如图：
![Screenshot 2024-10-25 at 21-06-56 Hello Theme Async VitePress](https://github.com/user-attachments/assets/7dddb3ec-d525-40cd-81f3-48f0717c087a)
只是不知为何运行npm run build并未生成静态网站的根目录？
是不支持npm run build命令吗？那么如何生成静态网站的根目录？

谢谢回复

**Comment by brightmann** [comment]
哈哈。原来在这里：~/at-vp-site/.vitepress/dist
